static/open-specs/liveobjects.yaml

```diff
@@ -0,0 +1,1386 @@
+openapi: 3.0.0
+info:
+  title: Ably LiveObjects REST API
+  version: 1.0.0
+  description: |
+    # LiveObjects API
+
+    LiveObjects provides a set of purpose-built APIs and data structures to handle the complexities of persisting and synchronizing state, freeing you to focus on building features instead of managing concurrency or conflict resolution.
+
+    LiveObjects enables you to store data as "objects" on a channel. These objects are automatically synchronized in realtime across all connected clients, and any conflicts that arise from concurrent updates are seamlessly resolved in the background.
+servers:
+  - url: https://main.realtime.ably.net/
+    description: Production server for Ably REST API
+
+paths:
+  /channels/{channelId}/objects:
+    get:
+      summary: Get a tree of objects
+      description: |
+        Fetches the list of objects stored on the channel.<br><br>
+
+        In the list API, data values represent a concrete piece of data (a number, a string, etc) or a reference to another object. The key in the data value indicates the type that you can expect to receive in the value.<br><br>
+
+        ```json
+          { "data": { "number" : 4 }}
+          { "data": { "string" : "Ably Pub/Sub" }}
+          { "data": { "boolean" : true }}
+          { "data": { "bytes": "TGl2ZU9iamVjdHMgaXMgYXdlc29tZQo=", "encoding": "base64" }}
+          { "data": {
+            "objectId":
+              "counter:Nz1ZiNjqsDfkDjA61xarinqpWsqEGAAw2mzWWtvX2b8@1742481614000"
+          }}
+        ```
+
+        <br>Maps are made of entries, which are a user-defined key (e.g. `myMapKey`) and a data value. Counters are data values containing a number.
```

Comment:
these descriptions of data values, maps, and counters seem like they are the wrong way around. 

IMO it should be; 
- there are two types of objects; maps and counters
- maps have XYZ
- counters have ABC